cypress/support/Train.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.  
+ * Licensed under the MIT License.
+ */
+
+const scorerModal = require('./components/ScorerModal')
+const trainDialogsGrid = require('./components/TrainDialogsGrid')
+const editDialogModal = require('./components/EditDialogModal')
+const helpers = require('./Helpers.js')
+
+function Today() { return Cypress.moment().format("MM/DD/YYYY") }
+
+// Cypress Flaw: to get true global data it must be attached to the window object.
+window.trainingSummary = undefined
+window.expectedTrainGridRowCount = 9999
+
+export function CreateNewTrainDialog()
+{
+  cy.Train_CreateNewTrainDialog()
+  trainDialogsGrid.CreateNewTrainDialog()
+}
+
+export function TypeYourMessage(message)
+{
+  editDialogModal.TypeYourMessage(message)
+  cy.Train_TypeYourMessage(message)
+}
+
+export function SelectAction(expectedResponse, lastResponse)
+{
+  scorerModal.ClickAction(expectedResponse)
+  cy.Train_SelectAction(expectedResponse, lastResponse)
+}
+
+export function Save()
+{
+  editDialogModal.ClickSaveButton()
+  cy.Train_Save()
+  trainDialogsGrid.VerifyPageTitle()
+  
+  // Bug 1558: Spinner Needed After Save Train Dialog and Other Save Actions
```

Comment:
The purpose of this spinner would only be to make testing easier.  The user should be blocked from proceeding waiting on the save